easybuild/easyconfigs/i/IPython/IPython-5.8.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,231 @@
+easyblock = 'Bundle'
+
+name = 'IPython'
+version = '5.8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('ZeroMQ', '4.2.5'),
+    ('matplotlib', '2.2.3', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+# let sanity check fail when auto-downloaded dependencies are detected when installing extensions
+exts_download_dep_fail = True
```

Comment:
this can be removed when using `PythonBundle`, should still add `use_pip = True` though